daemon/daemon/daemon_config.go

```diff
@@ -67,6 +67,7 @@ type Config struct {
 
 	DryMode      bool // Do not create BPF maps, devices, ..
 	RestoreState bool // RestoreState restores the state from previous running daemons.
+	KeepConfig   bool // When RestoreState is set, it keeps endpoints' configuration.
```

Comment:
Reword this comment to something like "Keep configuration of existing endpoints when starting up"